EPRN.Waste.API/Services/JourneyService.cs

```diff
@@ -91,7 +98,23 @@ public async Task<string> GetWasteType(int journeyId)
 
         public async Task<WasteRecordStatusDto> GetWasteRecordStatus(int journeyId)
         {
-            return await _wasteRepository.GetWasteRecordStatus(journeyId);
+            var journey = await _wasteRepository.GetWasteJourneyById_FullModel(journeyId);
```

Comment:
Return DTOs from the repositories - this is so only the repository layer has any knowledge of the db based models. Also, when performing the db queries only return the fields necessary for the class that is being populated, to keep queries to a minimum

Reply:
This still hasn't been addressed